src/Common/src/System/Text/OSEncoding.Windows.cs

```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable enable
```

Comment:
We shouldn't have these, right? It's opted in globally for hte library using the csproj

Reply:
it is required (at least temporarily) because System.Diagnostics.Process uses this file as well but its project does not has this enabled yet

Reply:
Correct, we can do a pass once everything is annotated and try to delete this pragmas, unfortunately that is the only way now to annotate shared files.